sim/paladin/paladin.go

```diff
@@ -77,18 +81,27 @@ func (paladin *Paladin) Initialize() {
 	paladin.registerHammerOfWrath()
 	paladin.registerHolyWrath()
 	paladin.registerAvengingWrath()
+	paladin.registerAuraMastery()
+
+        if paladin.primaryPaladinAura == proto.PaladinAura_SanctityAura {
+            paladin.currentPaladinAura = core.SanctityAuraAura(paladin.GetCharacter())
+        }
 }
 
 func (paladin *Paladin) Reset(_ *core.Simulation) {
 }
 
 // maybe need to add stat dependencies
-func NewPaladin(character *core.Character, talentsStr string) *Paladin {
+func NewPaladin(character *core.Character, options *proto.Player, pallyAura proto.PaladinAura) *Paladin {
 	paladin := &Paladin{
 		Character: *character,
 		Talents:   &proto.PaladinTalents{},
 	}
-	core.FillTalentsProto(paladin.Talents.ProtoReflect(), talentsStr, TalentTreeSizes)
+	core.FillTalentsProto(paladin.Talents.ProtoReflect(), options.TalentsString, TalentTreeSizes)
+
+        if pallyAura == proto.PaladinAura_SanctityAura {
+            paladin.primaryPaladinAura = pallyAura
+        }
```

Comment:
Looks like the indentation is a bit off here too

Reply:
will fix

Reply:
will fix